src/output.cpp

```diff
@@ -2,7 +2,7 @@
 #include "bpftrace.h"
 #include "log.h"
 #include "utils.h"
-#include <async_event_types.h>
+#include <ast/async_event_types.h>
```

Comment:
```suggestion
#include "ast/async_event_types.h"
```
and order alphabeticlaly